examples/BUILD.bazel

```diff
@@ -34,10 +36,13 @@ ts_library(
 ts_library(
     name = "bar_ts_library",
     srcs = ["bar.ts"],
-    tsconfig = ":tsconfig.json",
+    tsconfig = ":tsconfig-e2e.json",
```

Comment:
this should probably use `@npm//:@types` when it is available to show how all types can be automatically discovered. bar.ts has
```
// Repro for #31, should automatically discover @types/node
import * as fs from 'fs';
```

Reply:
:+1:

Reply:
done